extensions/identity-hub-verifier/src/main/java/org/eclipse/dataspaceconnector/identityhub/verifier/DidJwtCredentialsVerifier.java

```diff
@@ -43,30 +43,32 @@ class DidJwtCredentialsVerifier implements JwtCredentialsVerifier {
         this.monitor = monitor;
     }
 
-    public boolean isSignedByIssuer(SignedJWT jwt) {
+    public Result<Void> isSignedByIssuer(SignedJWT jwt) {
         String issuer;
         try {
             issuer = jwt.getJWTClaimsSet().getIssuer();
         } catch (ParseException e) {
-            monitor.warning("Error parsing issuer from JWT", e);
-            return false;
+            var failureMessage = "Error parsing issuer from JWT";
+            monitor.warning(failureMessage, e);
```

Comment:
As we are handling over the decision what to do in case of invalid credentials to the caller, perhaps we should consider not logging any warnings at this level.